modules/microshift-nw-router-config-ports.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-router-ports_{context}"]
+= Configuring router ports
+
+You can control which ports your devices use by configuring the router ingress fields.
+
+.Prerequisites
+
+* You installed {microshift-short}.
+* You created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` port values in the `ingress.ports.http` and `ingress.ports.https` fields to the ports you want to use:
++
+.Example `config.yaml` router settings
+[source,yaml]
+----
+#....
+ingress:
+  ports: # <1>
+    http: 80 #
+    https: 443
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed
+  status: Managed # <2>
+# ....
+----
+<1> Default ports shown. Customizable. Valid values for both port entries are a single, unique port in the 1-65535 range. The values of the `ports.http` and `ports.https` fields cannot be the same.
+<2> The default value. `Managed` is required for the ingress ports to remain open.
++
+[TIP]
+====
+If you complete all the configurations that you need to make in the {microshift-short} `config.yaml` file at the same time, you can minimize system restarts.
+====
+
+//.Verification
+//TODO Yes, Pablo to provide example; see end-to-end tests
```

Comment:
This example is not specially complex but implies using external tools like curl. Do we need to keep it?

Reply:
example of curl a route in a cluster's pod:
```
1.
sh-4.4# curl https://edge1-default.apps.example.com:10443/ -k --resolve edge1-default.apps.example.com:10443:10.1.235.168
this a test!
2.
% oc -n openshift-ingress get svc
NAME TYPE CLUSTER-IP EXTERNAL-IP PORT(S) AGE
router-default LoadBalancer 10.43.21.230 10.1.235.168,192.168.124.153 10080:30448/TCP,10443:31863/TCP 113m
router-internal-default ClusterIP 10.43.88.186 80/TCP,443/TCP,1936/TCP 113m
```

Reply:
@pacevedom and @ShudiLi Only if we think a customer will actually verify in this way.